### PR TITLE
Use the fragment registry in the `debug:fragments` command

### PR DESCRIPTION
**Pull request by bytehead**
Preparation PR to fix the issue #6471 afterwards.

### CODE REVIEW

File: core-bundle/src/Command/DebugFragmentsCommand.php [hunk 1]
```diff
@@ -12,51 +12,48 @@
...
+        parent::__construct();
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $rows = [];
-        $identifiers = $this->identifiers;
+        $fragments = $this->registry->all();
+        $identifiers = array_keys($fragments);
         natsort($identifiers);
```
Comment (m-vo) [inline core-bundle/src/Command/DebugFragmentsCommand.php]:
```suggestion
        ksort($fragments, SORT_NATURAL);
```

File: core-bundle/src/Command/DebugFragmentsCommand.php [hunk 2]
```diff
@@ -12,51 +12,48 @@
...
 
         foreach ($identifiers as $identifier) {
```
Comment (m-vo) [inline core-bundle/src/Command/DebugFragmentsCommand.php]:
```suggestion
        foreach ($fragments as $identifier => $config) {
```

File: core-bundle/src/Command/DebugFragmentsCommand.php [hunk 3]
```diff
@@ -12,51 +12,48 @@
...
-            $attributes = $this->attributes[$identifier];
-            $controller = $attributes['debugController'] ?? $this->configs[$identifier]->getController();
+            $config = $fragments[$identifier];
```
Comment (m-vo) [inline core-bundle/src/Command/DebugFragmentsCommand.php]:
```suggestion
```

**Review by m-vo — DISMISSED** [review]
LGTM. I think I would rewrite the `foreach` loop to directly iterate over keys and values, though (see suggestions).

**Review by m-vo — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @bytehead.

**Pull request merged**